apis/kueue/v1beta1/workload_types.go

```diff
@@ -168,6 +170,17 @@ type PodSetAssignment struct {
 	//   cloud.provider.com/topology-block: block-1
 	//   cloud.provider.com/topology-rack: rack-2
 	//
+	// Example:
```

Comment:
add some info what this example what is supposed to show. Because we have another example above, and it is not clear for the user why we need one more example

Reply:
Done